packages/ckb-indexer/mirgation.md

```diff
@@ -0,0 +1,55 @@
+## Migrate from native indexer
+
+Lumos is used for execution at the end of the server, so we provide `@ckb-lumos/indexer` and @ckb-`lumos/sql-indexer`. While bringing flexibility, it also increases the complexity of using it, such as having to run the database locally, compile the code, etc.
+
+Now that lumos plans to support running on the web, `native indexer` and `sql-indexerd` cannot run on the web. So we marked `native-indexer` and `sql-indexer` as not recommended in version 0.17.0-rc5, and introduced `ckb-indexer` to support running on web side. in this version, we will remove `native-indexer` and `sql-indexer` directly, if you plan to upgrade If you are planning to upgrade your version of lumos, please refer to this upgrade guide.
+
+### **Start Indexer**
+
+In `native-indexer`, you need to call `indexer.startForever();` . But in the `@ckb-lumos/ckb-indexer`, you don't need to call start anymore, you just need to create an indexer instance and we will turn it on for you automatically.
```

Comment:
```suggestion
Instead of calling `indexer.startForever()` in `native-indexer`, you just need to create an indexer instance in `@ckb-lumos/ckb-indexer`, and we will turn it on for you automatically.
```